alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -288,6 +296,32 @@ class AlertIndices(
         )
     }
 
+    private fun rolloverFindingIndex() {
+        val request = RolloverRequest(AlertIndices.FINDING_INDEX, null)
+        request.createIndexRequest.index(AlertIndices.FINDING_INDEX_PATTERN)
+            .mapping(AlertIndices.MAPPING_TYPE, AlertIndices.alertMapping(), XContentType.JSON)
+            .settings(Settings.builder().put("index.hidden", true).build())
+        request.addMaxIndexDocsCondition(findingMaxDocs)
+        request.addMaxIndexAgeCondition(findingMaxAge)
+        client.admin().indices().rolloverIndex(
+            request,
+            object : ActionListener<RolloverResponse> {
+                override fun onResponse(response: RolloverResponse) {
+                    if (!response.isRolledOver) {
+                        AlertIndices.logger.info(
+                            "${AlertIndices.HISTORY_WRITE_INDEX} not rolled over. Conditions were: ${response.conditionStatus}"
```

Comment:
`${AlertIndices.HISTORY_WRITE_INDEX}` should be `${FINDINGS_WRITE_INDEX}` instead that you will create